pylib/cqlshlib/cql3handling.py

```diff
@@ -1501,6 +1505,32 @@ def alter_type_field_completer(ctxt, cass):
                        ;
 '''
 
+syntax_rules += r'''
+<slaName> ::= <identifier>
+             | <quotedName>
+             | <unreservedKeyword>
+             ;
+
+<createSlaStatement> ::= "CREATE" "SERVICE_LEVEL" <slaName>
+                              ( "WITH" <slaProperty> ("AND" <slaProperty>)*)?
+                        ;
+
+<alterSlaStatement> ::= "ALTER" "SERVICE_LEVEL" ("IF" "EXISTS")? <slaName>
+                              ( "WITH" <slaProperty> ("AND" <slaProperty>)*)?
+                       ;
+
+<slaProperty> ::= "WORKLOAD_TYPE" "=" <stringLiteral>
+                 | "TIMEOUT" "=" <mapLiteral>
+                 | "SHARES" "=" <boolean>
+                 ;
+
+<dropSlaStatement> ::= "DROP" "SERVICE_LEVEL" ("IF" "EXISTS")? <slaName>
+                      ;
+
+<listSlaStatement> ::= "LIST" "SERVICE_LEVELS"
+                       ;
+'''
```

Comment:
another options: 
`ATTACH SERVICE_LEVEL <service_level_name> TO <role_name>;`

`LIST ALL ATTACHED SERVICE_LEVELS;`

`LIST ATTACHED SERVICE_LEVEL OF <role_name>;`

`DETACH SERVICE_LEVEL FROM <role_name>;`

Reply:
added those